README.md

```diff
@@ -1,17 +1,77 @@
-<p align="center">
-    <img src="https://raw.githubusercontent.com/Quansight/ragna/main/docs/assets/brand/logo-lockup-vertical/logo-lockup-vertical.png" width=30%>
-</p>
+<!-- Logo: Original in light background and white-logo in dark background -->
+<picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/Quansight/ragna/main/docs/assets/brand/logo-lockup-vertical/logo-lockup-vertical-white.png">
+    <img src="https://raw.githubusercontent.com/Quansight/ragna/main/docs/assets/brand/logo-lockup-vertical/logo-lockup-vertical.png" alt="Ragna logo" width=30%/>
+</picture>
 
-<hr>
+A RAG orchestration framework.
 
-A RAG orchestration framework. ⛵️
+---
 
-## Local development
+<!-- Badge table with useful links -->
+<table>
+<tr>
+    <th>Project</th>
+    <td>
+        <a href="https://pypi.org/project/Ragna/">
+            <img src="https://img.shields.io/pypi/v/ragna?colorA=1F2636&colorB=DF5538"
+         alt="PyPI" />
+        </a>
+    </td>
+    <td>
+        <a href="https://ragna.chat">
+            <img src="https://img.shields.io/badge/documentation-ragna.chat-gray.svg?colorA=1F2636&colorB=DF5538"
+         alt="Documentation" />
+        </a>
+    </td>
+</tr>
+<tr>
+    <th>Community</th>
+    <td>
+        <a href="https://github.com/Quansight.ragna/discussions">
+            <img src="https://img.shields.io/badge/support-GitHub_discussions-gray.svg?colorA=1F2636&colorB=3C8D89"
+         alt="GH Discussions" />
+        </a>
+    </td>
+    <td>
+        <a href="https://github.com/Quansight.ragna/issues/new/choose/">
+            <img src="https://img.shields.io/badge/bugs/features-GitHub_issues-gray.svg?colorA=1F2636&colorB=3C8D89"
+         alt="Issues" />
+        </a>
+    </td>
+</tr>
+<tr>
+    <th>CI</th>
+    <td>
+        <a href="https://github.com/Quansight/ragna/actions/workflows/test.yml">
+            <img src="https://github.com/Quansight/ragna/actions/workflows/test.yml/badge.svg"
+         alt="Tests" />
+        </a>
+    </td>
+    <td>
+        <a href="https://readthedocs.org/projects/ragna/">
+            <img src="https://img.shields.io/readthedocs/ragna"
+         alt="Docs build" />
+        </a>
+    </td>
```

Comment:
Let's not. CI badges are kinda useless. They don't give any information to the user and for maintainers, an issue is much better. Plus, we currently don't have any CI running on `main`, so this doesn't actually do anything.

```suggestion
```